SQL/0000-00-00-schema.sql

```diff
@@ -2014,3 +2016,28 @@ CREATE TABLE `feedback_mri_comments` (
   CONSTRAINT `FK_feedback_mri_comments_2` FOREIGN KEY (`PredefinedCommentID`) REFERENCES `feedback_mri_predefined_comments` (`PredefinedCommentID`),
   CONSTRAINT `FK_feedback_mri_comments_3` FOREIGN KEY (`FileID`) REFERENCES `files` (`FileID`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+CREATE TABLE `visit` (
+  `VisitID` int(10) unsigned NOT NULL auto_increment,
+  `VisitLabel` varchar(255) NOT NULL,
+  `VisitLegacyLabel` varchar(255) DEFAULT NULL,
+  CONSTRAINT `visit_PK` PRIMARY KEY (`VisitID`),
+  CONSTRAINT `visit_label_UK` UNIQUE KEY (`VisitLabel`),
+  CONSTRAINT `visit_legacy_label_UK` UNIQUE KEY (`VisitLegacyLabel`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+CREATE TABLE `visit_project_subproject_rel` (
+  `VisitProjetSubProjetID` int unsigned NOT NULL auto_increment,
```

Comment:
`VisitProjectSubprojectID` (lowercase p in Subproject, missing c in both spellings of project..)